src/commands/disconnectRepo/IDisconnectRepoContext.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ExecuteActivityContext, ISubscriptionActionContext } from "@microsoft/vscode-azext-utils";
+import { AzureSubscription } from "@microsoft/vscode-azureresources-api";
+import { IGitHubContext } from "../../gitHub/IGitHubContext";
+import { ContainerAppModel } from "../../tree/ContainerAppItem";
+
+export interface IDisconnectRepoContext extends ISubscriptionActionContext, IGitHubContext, ExecuteActivityContext {
+    targetContainer: ContainerAppModel;
+    subscription: AzureSubscription;
```

Comment:
It's really weird that `ISubscriptionActionContext` doesn't contain an `AzureSubscription`...

Reply:
Yeah we need to revisit this pattern in general so we don't need to keep adding double subscriptions

Reply:
Double...

https://www.youtube.com/watch?v=OQSNhk5ICTI